### PR TITLE
translate why-cliquet into English

### PR DESCRIPTION
**Pull request by phrawzty**
:eyes: :mag: :page_facing_up:

### CODE REVIEW

File: content/2015.07.cliquet.rationale-en.rst [hunk 1]
```diff
@@ -0,0 +1,494 @@
+Why did we make Cliquet?
```
Comment (Natim) [inline content/2015.07.cliquet.rationale-en.rst]: Why are we making Cliquet?

File: content/2015.07.cliquet.rationale-en.rst [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+:url: why-cliquet
+:slug: post-cliquet-rationale
+:date: 2015-07-14
+:summary:
+    Based on Pyramid, Cliquet is a project that allows you to concentrate on
+    the essentials when building APIs.
+
+.. note::
+
+    Translated from the French by `Dan Phrawzty <https://twitter.com/phrawzty>`_.
```
Comment (Natim) [inline content/2015.07.cliquet.rationale-en.rst:14]: Is it something we want to do on all translation? Should I add a note in the post I translated?

File: content/2015.07.cliquet.rationale-en.rst [hunk 3]
```diff
@@ -0,0 +1,494 @@
...
+
+The objective for the first trimester of 2015 was to build a service for
+storage and the `synchronisation of a reading list
+<{filename}2015.04.service-de-nuages.rst>`_. At the outset of the project we
+tried assembling all of the best practices and recommendations from different
+teams and currently deploy projects. Also, we wanted to take advantage of the
+*Firefox Sync* protocol for offline sync since it's robust and proven.
+
+Rather than write `yet another <http://blog.octo.com/en/design-a-rest-api/>`_
+`article <http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api>`_
```
Comment (Natim) [inline content/2015.07.cliquet.rationale-en.rst]: yet another ~article~~ solution?

Comment (Natim) [inline content/2015.07.cliquet.rationale-en.rst]: Never mind

File: content/2015.07.cliquet.rationale-en.rst [hunk 4]
```diff
@@ -0,0 +1,494 @@
...
+general.
+
+An important tenet in all of our projects is that of flexibility. This means
+that we try to make things optional where we can (and malleable where we
+can't) - Cliquet is no different. *Everything is configurable* right from the
+``.ini`` file! What's more, all of the projects that use the toolkit are
+deployed in exactly the same way - only the *configuration* is different.
+
+.. image:: {filename}/images/cliquet-notes-whiteboard.jpg
+    :alt: Une réunion à Paris...
```
Comment (Natim) [inline content/2015.07.cliquet.rationale-en.rst]: Shall we translate alternate as well?

File: content/2015.07.cliquet.rationale-en.rst [hunk 5]
```diff
@@ -0,0 +1,494 @@
...
+    class Bookmark(resource.BaseResource):
+        mapping = BookmarkSchema()
+
+        def process_record(self, new, old=None):
+            if old is not None and new['device'] != old['device']:
+                device = self.request.headers.get('User-Agent')
+                new['device'] = device
+            return new
+
+For more information, see the `resource docmentation
```
Comment (Natim) [inline content/2015.07.cliquet.rationale-en.rst]: documentation

**Comment by Natim** [comment]
Great thanks.

File: content/2015.07.cliquet.rationale-en.rst [hunk 6]
```diff
@@ -0,0 +1,494 @@
...
+practices in terms of production use and the HTTP protocol.**
+
+The Origins
+===========
+
+The objective for the first trimester of 2015 was to build a service for
+storage and the `synchronisation of a reading list
+<{filename}2015.04.service-de-nuages.rst>`_. At the outset of the project we
+tried assembling all of the best practices and recommendations from different
+teams and currently deploy projects. Also, we wanted to take advantage of the
```
Comment (almet) [inline content/2015.07.cliquet.rationale-en.rst]: deployed?

File: content/2015.07.cliquet.rationale-en.rst (hunk 2 above)
Comment (phrawzty) [inline content/2015.07.cliquet.rationale-en.rst:14]: It's important to note when an article has been translated, imho. I would suggest that you add a similar note on the post you translated as well, yes.

File: content/2015.07.cliquet.rationale-en.rst (hunk 1 above)
Comment (phrawzty) [inline content/2015.07.cliquet.rationale-en.rst]: This is a direct translation from the original French title "Pourquoi avons-nous fait Cliquet ?" which is _passé composé_. If the previous title is incorrect then we should change both. :smile:

Comment (Natim) [inline content/2015.07.cliquet.rationale-en.rst]: Also note that the french verbe `faire` is often use for diverse things other than `make`. IMHO, if you use build instead it should do the trick.

File: content/2015.07.cliquet.rationale-en.rst (hunk 4 above)
Comment (phrawzty) [inline content/2015.07.cliquet.rationale-en.rst]: Yes - I just forgot this one. :blush:

File: content/2015.07.cliquet.rationale-en.rst (hunk 1 above)
Comment (phrawzty) [inline content/2015.07.cliquet.rationale-en.rst]: How about going in a different direction? "Making the case for Cliquet", for example? It doesn't necessarily need to be a literal translation...

Comment (Natim) [inline content/2015.07.cliquet.rationale-en.rst]: Exactly :+1:

File: content/2015.07.cliquet.rationale-en.rst (hunk 2 above)
Comment (Natim) [inline content/2015.07.cliquet.rationale-en.rst:14]: Done.

Comment (almet) [inline content/2015.07.cliquet.rationale-en.rst:14]: Noting that something was translated makes sense, stating who did it with a link on their twitter account isn't useful IMO.

Comment (phrawzty) [inline content/2015.07.cliquet.rationale-en.rst:14]: I was following the lead set by @leplatrem and @ametaireau (according to `git blame`) in [this article](http://www.servicedenuages.fr/en/generic-storage-ecosystem).

Comment (almet) [inline content/2015.07.cliquet.rationale-en.rst:14]: No worries. I think there is a difference doing it for external contributors and for us, but I might be wrong here.

**Pull request merged**